client-src/elements/chromedash-report-external-reviews-page.ts

```diff
@@ -1,23 +1,23 @@
 // @ts-check
 import {Task} from '@lit/task';
 import '@shoelace-style/shoelace';
+import {
+  DefaultApiInterface,
+  OutstandingReview,
+  OutstandingReviewCurrentStageEnum as Stage,
```

Comment:
`Stage` sounds like an object that respresents a stage.  In fact, we have such objects used in this app.  This is actually an enum, so call it something like `StageEnum`.

Reply:
Good point. thanks!